moj_analytics/auth0_client.py

```diff
@@ -278,9 +282,15 @@ def add_users(self, users):
         if 'error' in response:
             raise AddGroupMemberError(response['message'])
 
-        print('Added users({users}) to group({group})'.format(
-            users=', '.join(user['email'] for user in users),
-            group=self['name']))
+    def delete_users(self, users):
+        response = self.api.request(
+            'DELETE',
+            'groups/{_id}/members'.format(**self),
```

Comment:
Should this be changed to `f'groups/{self._id}/members',`?

Reply:
Ta 👍